controllers/rollingupgrade_controller.go

```diff
@@ -151,35 +151,38 @@ func (r *RollingUpgradeReconciler) preDrainHelper(ruObj *upgrademgrv1alpha1.Roll
 // Operates on any scripts that were provided after the draining of the node.
 // kubeCtlCall is provided as an argument to decouple the method from the actual kubectl call
 func (r *RollingUpgradeReconciler) postDrainHelper(ruObj *upgrademgrv1alpha1.RollingUpgrade, nodeName string, kubeCtlCall string) error {
+	logr := r.logger(ruObj)
```

Comment:
Can you define this globally, so that we don't need this line in every method?

Reply:
Need to maintain some context then for the `RollingUpgrade`. Is it fine with you?